### PR TITLE
Task1 branch

### PR DESCRIPTION
**Pull request by IrynaSemenenko**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,11 @@
 # Log file
 *.log
 testIgnore/
+Projects/
```
Comment (Hmeliumm) [inline .gitignore:7]: Why this is needed?

File: .gitignore [hunk 2]
```diff
@@ -4,6 +4,11 @@
 # Log file
 *.log
 testIgnore/
```
Comment (Hmeliumm) [inline .gitignore:6]: Please delete, if it's not needed anymore

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (IrynaSemenenko) [reply .gitignore:7]: Because I want to create a new project for my homework.

File: .gitignore (hunk 2 above)
Comment (IrynaSemenenko) [reply .gitignore]: Done

File: .gitignore (hunk 1 above)
Comment (Hmeliumm) [reply .gitignore:7]: So you don't need to ignore this folder

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (IrynaSemenenko) [reply .gitignore]: I understood.  I deleted the "project" folder and added the "out" folder in the gitignore file.

**Review by Hmeliumm — APPROVED** [review]

**Pull request merged**